### ISSUE TITLE
Personenbild als Parameter aus der Mediathek festlegen

### ISSUE THREAD

**Issue by xwolfde** [RRZE-Webteam/rrze-faudir#55]
Um ein Personenbild darzustellen, soll es die Option geben, im Shortcode als auch im Block ein Bild aus der Mediathek auszuwählen, welches als Bild verwendet wird.

Als Parameter für den Shortcode wäre  `image=""` geeignet, wobei der Wert von` image=""` die WordPress-Instanz-spezifische Id des Bildes wäre. 
Ausdrücklich soll kein Slug und keine URL als Wert zugelassen werden, da dies in anderen Plugins bereits zu Problemen führte.

Dies soll bereits vor und unabhängig von Version 2 gehen, bei der mit Hilfe eines Custom Post Types ein Bild definiert werden könnte. Falls via Parameter/Block-Option ein Bild aus der Mediathek verwendet wurde und im Custom Post Type ein anderes Bild als Artikelbild festgelegt wurde, wird das Bild aus dem Parameter/Block-Option verwendet.

**Comment by debokshi** [comment]
@xwolfde Just to make sure this is only for the page format(when there is only one person visible), right?

**Comment by xwolfde** [comment]
Yes, but its for every display, where a single person is addressed with ids or name. Not only for the page frmat, but also for other form,ats with a single person. E.g. card-format.


The idea is, that in some cases we want to display a person with another image as the one, which was defined as thumbnail image in the custom post type.

**Comment by xwolfde** [comment]
Funktioniert
